packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -68,6 +66,38 @@ import { useHandleClaim } from '../ProtocolDetails/useHandleClaim';
+            <SizableText size="$bodySm" color={item.title.color}>
+              {item.title.text}
+            </SizableText>
+            <SizableText
+              size="$bodySm"
+              color={item.description.color || '$textSubdued'}
+            >
+              {item.description.text}
+            </SizableText>
+          </XStack>
+          {index !== managers.items.length - 1 ? (
+            <XStack w="$4" h="$4" ai="center" jc="center">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)